java/test/src/main/java/org/ray/api/benchmark/MicroBenchmarks.java

```diff
@@ -0,0 +1,48 @@
+package org.ray.api.benchmark;
+
+import org.ray.api.Ray;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MicroBenchmarks {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MicroBenchmarks.class);
+
+  public static Object simpleFunction() {
+    return null;
+  }
+
+  private static void time(Runnable runnable, int numRepeats, String name) {
+    LOGGER.info("Benchmark \"{}\" started.", name);
+    long start = System.nanoTime();
```

Comment:
final

Reply:
ok, but I think for small functions, `final` doesn't make much difference.